HW03/Alexey_Zalyotov/lib/task.rb

```diff
@@ -0,0 +1,24 @@
+class Task
+  OPEN = :open
+
+  attr_reader :task_body, :repo, :task_id
+  attr_accessor :solution, :status, :review_comment
+
+  def initialize(task_body:, repo:, task_id:)
+    @task_body = task_body
+    @repo = repo
+    @task_id = task_id
```

Comment:
I think we don't need to use `task` prefix, it's obvious from the class definition  
```suggestion
    @body = body
    @repo = repo
    @id = task_id
```

Reply:
You`re right. Done